gtsfm/densify/mvs_base.py

```diff
@@ -39,9 +43,12 @@ def densify(self, images: Dict[int, Image], sfm_result: GtsfmData) -> Tuple[np.n
                 with shape (N, 3) where N is the number of points
             dense_point_colors: RGB color of each point in the dense point cloud
                 with shape (N, 3) where N is the number of points
+            densify_metrics: Metrics group containing metrics for dense reconstruction
         """
 
-    def create_computation_graph(self, images_graph: Delayed, sfm_result_graph: Delayed) -> Tuple[Delayed, Delayed]:
+    def create_computation_graph(
+        self, images_graph: Delayed, sfm_result_graph: Delayed
+    ) -> Tuple[Delayed, Delayed, Delayed, Delayed]:
```

Comment:
thanks for the PR, @codyly. Do you mind updating the docstring to explain the 4 return types now?

Reply:
marking as resolved, thanks